Description: Turn off debugging by default. Logging to syslog will slow down loading.

Index: gdm-3.5.4.2+git20120803.00576a2d/common/gdm-common.c
===================================================================
--- gdm-3.5.4.2+git20120803.00576a2d.orig/common/gdm-common.c	2012-08-06 19:19:12.000000000 +0200
+++ gdm-3.5.4.2+git20120803.00576a2d/common/gdm-common.c	2012-08-06 19:20:26.000000000 +0200
@@ -47,6 +47,7 @@
 gboolean
 gdm_is_version_unstable (void)
 {
+#if 0
         char   **versions;
         gboolean unstable;
 
@@ -63,6 +64,9 @@
         g_strfreev (versions);
 
         return unstable;
+#else
+        return FALSE;
+#endif
 }
 
 gboolean
